tests/framework/checks/skips.go

```diff
@@ -162,6 +162,18 @@ func SkipIfUseFlannel(virtClient kubecli.KubevirtClient) {
 	}
 }
 
+func SkipIfPrometheusNotEnabled(virtClient kubecli.KubevirtClient) {
```

Comment:
Please remove this as you already have https://github.com/kubevirt/kubevirt/blob/7d82cd6a050a4b31d88405552e2384a38d66b89d/tests/decorators/decorators.go#L13

Reply:
+1, already spoke offline about that

Reply:
removed